Code/Tools/ProjectManager/Source/Application.cpp

```diff
@@ -111,6 +111,14 @@ namespace O3DE::ProjectManager
             }
         }
 
+        m_settings = AZStd::make_unique<Settings>();
+        AZ_Assert(m_settings, "Failed to create Settings");
+
+        if (!m_settings->IsInitialized())
+        {
+            return false;
```

Comment:
If we're going to bail out here in the extremely rare case m_settings is not initialized, we should notify the user via message box and/or AZ_Error so the specific error ends up in the logs.

Reply:
That makes sense, I'll remove it.

Reply:
I suppose if you wanted to you could pass m_settingsRegistry from Application to SettingsInterface when it is created, rather than having SettingsInterface access the singleton directly.